app/apps/addresses/models.py

```diff
@@ -121,7 +123,25 @@ def search_and_set_bag_nummeraanduiding_id(self):
                 "_embedded", {}
             ).get("nummeraanduidingen", [])
         except Exception:
-            bag_search_nummeraanduidingen = []
+            pass
+
+        # If no bag_search_nummeraanduidingen is found, try to search for BAG with address params.
+        if not len(bag_search_nummeraanduidingen) and self.street_name:
+            try:
```

Comment:
Om een lege list te checken hoef je `len()` niet te gebruiken:
`if not bag_search_nummeraanduidingen ...`